app/templates/cli.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+'use strict';
+var meow = require('meow');
+var <%= camelModuleName %> = require('./');
+
+var cli = meow({
+	help: [
+		'Usage',
+		'  $ <%= moduleName %> [object]',
+		'',
+		'Examples',
+		'  $ <%= moduleName %>',
+		'  unicorns & rainbows',
+		'',
+		'  $ <%= moduleName %> ponies',
+		'  ponies & rainbows'
```

Comment:
Can you add a option section for: https://github.com/sindresorhus/generator-nm/blob/master/app/templates/readme.md#foo